ui/src/app/machines/views/MachineDetails/MachineStorage/FilesystemsTable/FilesystemsTable.test.tsx

```diff
@@ -157,15 +157,70 @@ describe("FilesystemsTable", () => {
     expect(wrapper.find("AddSpecialFilesystem").exists()).toBe(true);
   });
 
-  it("can show an action to remove a filesystem", () => {
+  it("can remove a disk's filesystem", () => {
```

Comment:
Driveby rewrite of tests for each type of filesystem removal, including action assertion